{"project": "fastapi", "bug": 12, "project_url": "https://github.com/tiangolo/fastapi", "file_changed": "fastapi/security/http.py", "buggy_commit_id": "d61f5e4b555b123bf222503fc0e076cbae6a7ebc", "fixed_commit_id": "d262f6e9296993e528e2327f0a73f7bf5514e7c6", "lines_deleted": {"114": "            raise HTTPException(\n", "115": "                status_code=HTTP_403_FORBIDDEN,\n", "116": "                detail=\"Invalid authentication credentials\",\n", "117": "            )\n"}, "lines_added": {"114": "            if self.auto_error:\n", "115": "                raise HTTPException(\n", "116": "                    status_code=HTTP_403_FORBIDDEN,\n", "117": "                    detail=\"Invalid authentication credentials\",\n", "118": "                )\n", "119": "            else:\n", "120": "                return None\n"}}